.github/workflows/ci.yml

```diff
@@ -21,3 +21,14 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
       - name: Run linting
         run: make lint
```

Comment:
I think we are going to want to cache the docker builds
https://dev.to/dtinth/caching-docker-builds-in-github-actions-which-approach-is-the-fastest-a-research-18ei